test/Pantry-test.js

```diff
@@ -34,19 +34,18 @@ describe('Pantry', () => {
   it('should return missing ingredients and amount for a given recipe', () => {
     const recipe1 = sampleData[0];
     const recipe2 = sampleData[1];
-    expect(user.pantry.getMissingIngredients(recipe1).to.deep.equal([]))
-    expect(user.pantry.getMissingIngredients(recipe2).to.deep.equal([
+    expect(user.pantry.getMissingIngredients(recipe1)).to.deep.equal([]);
+    expect(user.pantry.getMissingIngredients(recipe2)).to.deep.equal([
       {'ingredient': 1009016, 'amount': 1.5},
       {'ingredient': 20027, 'amount': 1},
       {'ingredient': 1002046, 'amount': 1},
-      {'ingredient': 20027, 'amount': 1},
```

Comment:
This was causing a headache. Finally realized we accidentally put it in there twice. Deleted one and viola! It worked.